cpp/src/spatial/hausdorff.cu

```diff
@@ -122,15 +111,15 @@ struct hausdorff_functor
 
     template<typename T>
     std::enable_if_t<std::is_floating_point<T>::value, std::unique_ptr<cudf::column>>
-    operator()(cudf::column_view const& x,
-               cudf::column_view const& y,
-               cudf::column_view const& trajectory_lengths,
+    operator()(cudf::column_view const& xs,
```

Comment:
What is the `s` for?  What is a `space`? Are these not trajectories?

Reply:
Hausdorff distance is the maximum of the minimum distance between pairs of points in two spaces. Trajectories may have been the original use case for our implementation, but are misnomers in regard to the mathematical definition of the algorithm.

Reply:
“xs” was just plural for “x”. It gave me the option to use “x” for a single value later in the function, but I ended io kot using it. In languages like Haskell, list-operation functions often use x and xs as the head (singular) and tail (plural) arguments. I’m happy to use “x” and “y” if thats preferable.

Reply:
I personally prefer xs and ys as an idiom, though, because for functions which will use x and y as variable names, it eliminates the naming conflict.